pkgs/top-level/aliases.nix

```diff
@@ -187,6 +187,7 @@ mapAliases ({
   dagger = throw "'dagger' has been removed from nixpkgs, as the trademark policy of the upstream project is incompatible"; # Added 2023-10-16
   dart_stable = dart; # Added 2020-01-15
   dat = nodePackages.dat;
+  deadcode = throw "'deadcode' has been removed from nixpkgs as upstream is unmaintained. The official go/x/tools/deadcode is part of the 'gotools' package"; # Added 2023-10-09
```

Comment:
```suggestion
  deadcode = throw "'deadcode' has been removed, as upstream is abandoned since 2019-04-27. Use the official deadcode from 'gotools' package."; # Added 2023-10-09
```